scripts/ralph/overnight-prd154-156.sh

```
@@ -0,0 +1,588 @@
+  echo -e "\n${YELLOW}=== Usage limit — waiting for reset ===${NC}"
+  countdown "$sleep_duration" "Limit wait..."
+  echo -e "${GREEN}Resuming...${NC}"
+}
+
+# --- one headless claude invocation ------------------------------------------
+# Sets: CLAUDE_EXIT, CLAUDE_OUTPUT, CLAUDE_RESULT
+run_claude() {
+  local wt="$1" prompt_file="$2" tmo="$3" logfile="$4"
+  local tmp; tmp=$(mktemp)
+
+  cd "$wt"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)